src/kernel/coinstats.cpp

```diff
@@ -4,16 +4,32 @@
 
 #include <kernel/coinstats.h>
 
+#include <chain.h>
 #include <coins.h>
 #include <crypto/muhash.h>
 #include <hash.h>
+#include <node/blockstorage.h>
+#include <primitives/transaction.h>
+#include <script/script.h>
 #include <serialize.h>
+#include <span.h>
+#include <streams.h>
+#include <sync.h>
+#include <tinyformat.h>
 #include <uint256.h>
+#include <util/check.h>
 #include <util/overflow.h>
 #include <util/system.h>
 #include <validation.h>
+#include <version.h>
 
+#include <assert.h>
```

Comment:
We want to use/are migrating to the C++ headers. I know IWYU suggests `<assert.h>` (have been meaning to make it suggest the newer headers), but it will be equally as happy with [`<cassert>`](https://en.cppreference.com/w/cpp/error/assert). Thanks for fixing these up in any case.